docs/architecture/smart-rollups.mdx

```diff
@@ -1,266 +1,192 @@
 ---
-title: Smart Optimistic Rollups
-authors: 'Nomadic Labs, TriliTech'
+title: Smart Rollups
+authors: 'Nomadic Labs, TriliTech, Tim McMackin'
 last_update:
-  date: 30 June 2023
+  date: 4 December 2023
 ---
 
-Rollups play a crucial part in providing next-generation scaling on Tezos. This page gives a technical introduction to Smart Rollups, their optimistic nature, and an intro to developing your own WASM kernel.
+import LucidDiagram from '@site/src/components/LucidDiagram';
 
-## Examples
+Smart Rollups play a crucial part in providing high scalability on Tezos.
+They handle transactions and logic in a separate environment that doesn't need to follow all of the same rules as the main Tezos network.
+For this reason, they are not subject to the same block delays and data space constraints as the main Tezos network.
 
-For examples of Smart Rollups, see this repository: https://gitlab.com/tezos/kernel-gallery.
+The transactions and logic that Smart Rollups run is called _layer 2_ to differentiate it from the main network, which is called _layer 1_.
+
+The code of a Smart Rollup is public, and anyone can run a Smart Rollup node to run that code and verify that other nodes are running it correctly, just like anyone can run nodes, bakers, and accusers on layer 1.
+This code, called the _kernel_, must behave in a deterministic manner and use a proof-generating virtual machine (PVM) to generate a proof of its state.
+In this way, multiple nodes can run the same code to create a single state for the rollup, and other nodes can verify the state of the rollup.
+
+For a tutorial on Smart Rollups, see [Deploy a Smart Rollup](../tutorials/smart-rollup).
+
+For reference on Smart Rollups, see [Smart Optimistic Rollups](https://tezos.gitlab.io/active/smart_rollups.html) in the Octez documentation.
+
+This diagram shows a high-level view of how Smart Rollups interact with layer 1:
+
+<LucidDiagram width="640px" height="480px" src="https://lucid.app/documents/embedded/0502acd1-ac3a-45a5-b3a6-8b0d77d6e951" id="BE6RELfrN5xE" />
+
+## Uses for Smart Rollups
+
+Smart Rollups allow you to run large amounts of processing that would be impractical on layer 1.
+They also allow you to avoid some transaction fees and storage fees.
```

Comment:
> Smart Rollups allow you to run large amounts of processing that would be too slow or expensive to run on layer 1. Thus they allow you to avoid some transaction fees and storage fees.